docs/rules/prefer-json-parse-buffer.md

```diff
@@ -17,7 +17,7 @@ const packageJson = JSON.parse(await fs.readFile('./package.json', 'utf8'));
 
 ```js
 const promise = fs.readFile('./package.json', {encoding: 'utf8'});
-const packageJson = JSON.parse(promise);
+const packageJson = JSON.parse(await promise);
```

Comment:
Somehow missed this. This is not nessary forthis change, but since we are here, I just fixed it.